bundles/org.openhab.binding.bambulab/src/main/java/org/openhab/binding/bambulab/internal/BambuLabBindingConstants.java

```diff
@@ -55,6 +58,8 @@ public enum Channel {
         CHANNEL_TIME_LAPS("time-laps"),
         CHANNEL_USE_AMS("use-ams"),
         CHANNEL_VIBRATION_CALIBRATION("vibration-calibration"),
+        CHANNEL_CAMERA_RECORD("camera-record", true),
+        CHANNEL_CAMERA_IMAGE("camera-image"),
```

Comment:
Should have noticed this before. It would be better if these channel id's woudl be modelled as string constants and put into the constants file

Reply:
Why? They are still constants, but thanks to it being enum I can do extra things (like check if this channel supports command)

Reply:
I meant to keep the enum, but extract the channel to a string constant. But i totally missed this is already in the constants file and there is no additional beneft, So never mind.